e2e/framework/framework.go

```diff
@@ -76,6 +76,9 @@ func New(baseName string) *Framework {
 func (f *Framework) BeforeEach() {
 	var err error
 	By("Building a namespace api object")
+	annotations := make(map[string]string)
+	annotations["iam.gke.io/gcp-service-account"] = "external-secrets@external-secrets-operator.iam.gserviceaccount.com"
+	util.UpdateKubeSA("eso-external-secrets", f.KubeClientSet, "default", annotations)
```

Comment:
I am sure there is a better place to do this. Maybe from the CI script instead of here? For now I am doing here to test